.sentryclirc

```diff
@@ -0,0 +1 @@
+; this stub prevents global .sentryclirc from being loaded
```

Comment:
The `integration::info::command_info_no_token` test is still failing for be because it is picking up credentials from my global config.